pkg/services/dashboards/dashboard.go

```diff
@@ -83,4 +85,7 @@ type Store interface {
 	// CountDashboardsInFolder returns the number of dashboards associated with
 	// the given parent folder ID.
 	CountDashboardsInFolder(ctx context.Context, request *CountDashboardsInFolderRequest) (int64, error)
+
+	// Save a k8s resource directly
+	SaveK8sDashboard(ctx context.Context, orgID int64, uid string, meta metav1.ObjectMeta, body *simplejson.Json) (*Dashboard, error)
```

Comment:
@mildwonkey -- anything too cringy about exposing this here?   

Folder support seems off -- but will have to sort that out anyway